### PR TITLE
[REF] Minor Test clean up

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Minor cleanup within the test classes

Before
----------------------------------------
Less clean

After
----------------------------------------
More clean

Technical Details
----------------------------------------
Uses a trait for permissions functions, some comments cleanup, one unused function removed

Comments
----------------------------------------
Have at it jenkins - oh when you are ready of course

### CODE REVIEW

File: tests/phpunit/api/v3/ACLPermissionTest.php [hunk 1]
```diff
@@ -442,47 +443,6 @@ public function setUpEntities($entity) {
...
-   * All but first results returned.
-   * @implements CRM_Utils_Hook::aclWhereClause
-   * @param $type
-   * @param $tables
-   * @param $whereTables
-   * @param $contactID
-   * @param $where
-   */
-  public function aclWhereOnlySecond($type, &$tables, &$whereTables, &$contactID, &$where) {
-    $where = " contact_a.id > 1";
```
Comment (mattwire) [inline tests/phpunit/api/v3/ACLPermissionTest.php]: @eileenmcnaughton Is this function unused? As I can't see a replacement in the trait?

File: tests/phpunit/api/v3/ACLPermissionTest.php [hunk 2]
```diff
@@ -442,47 +443,6 @@ public function setUpEntities($entity) {
...
-   * Only specified contact returned.
-   * @implements CRM_Utils_Hook::aclWhereClause
-   * @param $type
-   * @param $tables
-   * @param $whereTables
-   * @param $contactID
-   * @param $where
-   */
-  public function aclWhereOnlyOne($type, &$tables, &$whereTables, &$contactID, &$where) {
-    $where = " contact_a.id = " . $this->allowedContactId;
```
Comment (mattwire) [inline tests/phpunit/api/v3/ACLPermissionTest.php]: @eileenmcnaughton Is this function unused? As I can't see a replacement in the trait?

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/ACLPermissionTest.php]: see above

**Comment by mattwire** [comment]
@eileenmcnaughton So tests are passing. Assuming you meant to remove two unused functions aclWhereOnlyOne and aclWhereOnlySecond this is ok to merge - please confirm that's correct first?

**Comment by eileenmcnaughton** [comment]
@mattwire yes, they are in the trait & the tests would have failed if there was a problem

**Pull request merged**